src/binaries/Cargo.toml

```diff
@@ -55,7 +55,7 @@ tonic = "0.7.2"
 tracing = "0.1.35"
 url = "2.2.2"
 
-openraft = { git = "https://github.com/datafuselabs/openraft", rev = "v0.7.1" }
+openraft = { git = "https://github.com/drmingdrmer/openraft", rev = "v0.7.2-alpha.1" }
```

Comment:
Is this expected?

Reply:
Yes. We need to update openraft frequently to some alpha version, which should not pollute the official repo tags.